### PR TITLE
US5562 - Add CKY Sites to MP

### PR DESCRIPTION
**Pull request by eymiha**
SQL Script to Add CKY sites Andover, Georgetown, Richmond and Downtown to MinistryPlatform. The pastor's contact and dp_user record is created for the Georgetown and Richmond sites, and Brian Tome is used as the pastor for Downtown.

### CODE REVIEW

File: CI/SQL/20161206_090000_US5562-addSitesToMP.sql [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+      ,[Publications_Manager]
+      ,[Contact_ID]
+      ,[User_GUID]
+      ,[Keep_For_Go_Live]
+      ,[Read_Permitted]
+      ,[Create_Permitted]
+      ,[Update_Permitted]
+      ,[Delete_Permitted]
+    ) VALUES (
+       'Brad Stone'
```
Comment (IngageKrizJ) [inline CI/SQL/20161206_090000_US5562-addSitesToMP.sql]: User_Name and User_Email on these should both be the email address (user_name is what is used for logging in).

File: CI/SQL/20161206_090000_US5562-addSitesToMP.sql [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+SET @name = 'Georgetown';
+PRINT @name;
+
+  -- assert pastor
+SELECT @pastor_id = User_ID FROM [dbo].[dp_Users]
+ WHERE [User_Email] = 'bstone@xroadschurch.org';
+IF @@ROWCOUNT = 0
+  BEGIN
+    PRINT '    inserting pastor for '+@name;
+    INSERT INTO [dbo].[Contacts] (
```
Comment (IngageKrizJ) [inline CI/SQL/20161206_090000_US5562-addSitesToMP.sql]: When inserting a contact, we should set email_address as well - that should be in sync with the dp_Users.User_Name and dp_Users.Email_Address.

Comment (eymiha) [inline CI/SQL/20161206_090000_US5562-addSitesToMP.sql]: Changed.

**Comment by eymiha** [comment]
It's interesting that the source of truth for a user's email is in Contacts, but the value is also replicated (but not asserted) in dp_User as User_Name **_and_** User_Email.

**Pull request merged**